client/hooks/authMethods.ts

```diff
@@ -0,0 +1,47 @@
+import toast from 'react-hot-toast';
+import { useRouter } from 'next/router';
+
+import { SignInUpFormValues } from 'shared/types';
+import { useAppDispatch, useAppSelector } from 'hooks/reduxSelector';
+import { reset, signIn, signOut, signUp } from 'redux/auth/authSlice';
+
+export const useAuthMethods = () => {
+  const { isError, error } = useAppSelector((state: any) => state.auth);
```

Comment:
State variable already has types.

```suggestion
  const { isError, error } = useAppSelector((state) => state.auth);
```